src/lightbox/LightboxContainer.js

```diff
@@ -7,6 +7,7 @@ import { connectActionSheet } from '@expo/react-native-action-sheet';
 import type { Actions, Auth, Message, ImageResource } from '../types';
 import connectWithActions from '../connectWithActions';
 import { getAuth } from '../selectors';
+import { getCurrentRealm } from '../account/accountSelectors';
```

Comment:
you can import `getCurrentRealm` from `../selectors` too.
As it exports all selectors.
https://github.com/zulip/zulip-mobile/blob/master/src/selectors.js